### ISSUE TITLE
Hardcoded mount points in the dockerr-compose.yml

### ISSUE THREAD

**Issue by chrisgorgo** [poldracklab/crn_deploy#1]
Currently `/srv/bids-core/persistent/data` and `/srv/crn-server/persistent` are hardcoded in the docker-compose.yml file. It would be more flexible it variables defined in `.env` were used. See https://docs.docker.com/compose/environment-variables/

It would be also good to explain what's the difference between `/srv/bids-core/persistent/data` and `/srv/crn-server/persistent`.

**Comment by constellates** [comment]
I'll double check but I don't believe it is possible to support the necessary behavior with environment variables. This path is meant to map a docker volume to a local filesystem volume. Docker environment variables are not available until you run `docker run` so they can't be used during building containers and mapping volumes.

I can add more documentation explaining how to modify the config.yml.

**Comment by chrisgorgo** [comment]
Have a look here:
https://docs.docker.com/compose/environment-variables/#/the-env-file

**Comment by constellates** [comment]
Thanks for the link. I've updated the repo to use the .env file to setup mount points and also choose what branch of the crn app and server repos to pull from. Both of these may change as we move the SFTP connection into the containers and move the app and server containers to docker hub, but for now they're at least in a .env config file. The readme has also been updated to include the .env file.
